presto-kudu/src/main/java/com/facebook/presto/kudu/properties/KuduTableProperties.java

```diff
@@ -66,6 +66,7 @@
     public static final String PARTITION_BY_RANGE_COLUMNS = "partition_by_range_columns";
```

Comment:
I would change cmt msg to:

```
Add Kudu table properties num_replicas and table_id
```

Reply:
I changed title to "Emit Kudu table property num_replicas in SHOW CREATE TABLE", as this table property is already accepted on creating a table, but it was not possible until now to show it in the output of `SHOW CREATE TABLE`.
The new proposed table property `table_id` has been removed, as it provide no real benefit. It would be ignored on table creation, as it is always set by Kudu.